kong/plugins/grpc-gateway/deco.lua

```diff
@@ -227,7 +227,10 @@ function deco:upstream(body)
   local body_variable = self.endpoint.body_variable
   if body_variable then
     if body and #body > 0 then
-      local body_decoded = decode_json(body)
+      local body_decoded, err = decode_json(body)
+      if err then
+        return nil, err
```

Comment:
```suggestion
        return nil, "decode json err: " .. err
```

Reply:
Thank you for your review. I will take this suggest.